buildenv/jenkins/omrbuild.groovy

```diff
@@ -20,26 +20,38 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-def setBuildStatus(String message, String state, String sha) {
-    context = "continuous-integration/eclipse-omr/branch/${params.BUILDSPEC}"
-    step([
-        $class: "GitHubCommitStatusSetter",
-        contextSource: [$class: "ManuallyEnteredCommitContextSource", context: context],
-        errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
-        commitShaSource: [$class: "ManuallyEnteredShaSource", sha: sha ],
-        statusBackrefSource: [$class: "ManuallyEnteredBackrefSource", backref: "${BUILD_URL}flowGraphTable/"],
-        statusResultSource: [$class: "ConditionalStatusResultSource", results: [[$class: "AnyBuildResult", message: message, state: state]] ]
-    ]);
-}
+defaultCompile = 'make -j4'
+defaultReference = '${HOME}/gitcache'
+autoconfBuildDir = '.'
+cmakeBuildDir = 'build'
+workspaceName = 'Build'
+scmVars = null
+
+dockerImage = null
+dockerImageName = (params.IMAGE_NAME) ? params.IMAGE_NAME : "buildomr"
+
+os = (params.OS) ? params.OS : "ubuntu20"
+arch = (params.ARCH) ? params.ARCH : "x86_64"
```

Comment:
Yes, it can be moved to `SPECS` as parameters to the generic script to generate a Dockerfile. For the time being, we neither have other Dockerfiles or a generic script. I can leave a comment to move these parameters into `SPECS` once multiple Dockerfiles are supported.

Reply:
Comment added.